### PR TITLE
Adjustments baby

### PR DESCRIPTION
**Pull request by KianMiridoozini**
Search :
Added icons for cocktail search
adjusted styling

Views:
Added 2 views (and updated the respective repository queries)

Adjusted:
scrollbar
Admin editing/deleting other users from their profile

Fixed:
Sort indicator
Messages
Register button

### CODE REVIEW

File: app/repositories/CocktailRepository.php [hunk 1]
```diff
@@ -202,18 +202,11 @@ public function getAllSortedByDate()
...
     public function getAllSortedByLikes()
     {
-        $stmt = $this->db->query("
-            SELECT c.*, COUNT(l.like_id) AS like_count
-            FROM cocktails c
-            LEFT JOIN likes l ON c.cocktail_id = l.cocktail_id
-            GROUP BY c.cocktail_id
-            ORDER BY like_count DESC
-        ");
+        $stmt = $this->db->query("SELECT * FROM view_cocktails_sorted_by_likes");
```
Comment (jayb2302) [inline app/repositories/CocktailRepository.php:205]: Nice

File: public/assets/css/components/search.css [hunk 2]
```diff
@@ -1,48 +1,111 @@
...
+  }
+  & .fa-magnifying-glass {
+    position: absolute;
+    top: 50%;
+    left: 10px;
+    transform: translateY(-50%);
+    color: var(--accent-primary);
+    pointer-events: none;
+  }
+
```
Comment (jayb2302) [inline public/assets/css/components/search.css:34]: Why do we need the icons absolute? I also tought Wed have icons style in icons.css

**Review by jayb2302 — APPROVED** [review]
Awesome!!

File: public/assets/css/components/search.css (hunk 2 above)
Comment (KianMiridoozini) [reply public/assets/css/components/search.css:34]: I did not add that. i just specified that the absolute you had implemented, only affects magnifying glass and not the new additions of other icons

**Pull request merged**